src/averageAges.js

```diff
@@ -20,6 +20,32 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const ages = [];
+  let averageAge = 0;
+  let eteration = 0;
+  const isMan = human => human.sex === 'm';
+
+  century
+    ? people.map((human) => {
+      if (isMan(human) && Math.ceil(human.died / 100) === century) {
```

Comment:
Consider using constants instead of magic numbers, it's an antipattern
```suggestion
      if (isMan(human) && Math.ceil(human.died / CENTURY) === century) {
```